src/sqlfluff/core/rules/std/L045.py

```diff
@@ -51,25 +51,12 @@ def _visit_sources(
         queries: Dict[str, List[SelectCrawler]],
     ):
         for select_info in select_info_list:
-            # Process nested SELECTs.
             for source in SelectCrawler.crawl(
                 select_info.select_statement, queries, dialect
             ):
                 if isinstance(source, list):
                     cls._visit_sources(source, dialect, queries)
 
-            # Process the query's sources.
-            for alias_info in select_info.select_info.table_aliases:
```

Comment:
After fixing the bug, the coverage report said this code was no longer being executed. It turns out to be unnecessary now. Fewer bugs and shorter code -- TDD and code coverage checks FTW!